### PR TITLE
chore: merge feature into master

### PR DESCRIPTION
**Pull request by MadCcc**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/timeline/index.en-US.md [hunk 1]
```diff
@@ -26,29 +26,37 @@ Vertical display timeline.
 <code src="./demo/label.tsx">Label</code>
 <code src="./demo/wireframe.tsx" debug>Wireframe</code>
 
-## API
+```__react
+import Alert from '../alert';
+ReactDOM.render(<Alert message="After version 5.3.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
```
Comment (yoyo837) [inline components/timeline/index.en-US.md:31]: 5.2.0 这个需要改 #40424里的, 另发PR吗

Comment (yoyo837) [reply components/timeline/index.en-US.md:31]: #40615

Comment (li-jia-nan) [reply components/timeline/index.en-US.md:31]: 这个写法要改一下，这么写是没有样式的，等合进去我明天早上单独开个 PR 搞一下吧

**Review by yoyo837 — APPROVED** [review]

**Comment by yoyo837** [comment]
不提醒你不干活吗? @github-actions 😂

**Comment by li-jia-nan** [comment]
我刚想说，这个怎么还要手动 approve 的，然后机器人就来了……

File: components/timeline/index.en-US.md (hunk 1 above)
Comment (MadCcc) [reply components/timeline/index.en-US.md]: 改

**Pull request merged**